src/strings/unicode_ops.c

```diff
@@ -108,13 +108,41 @@ MVMGrapheme32 MVM_unicode_lookup_by_name(MVMThreadContext *tc, MVMString *name)
     MVMuint64 size;
     char *cname = MVM_string_ascii_encode(tc, name, &size, 0);
     MVMUnicodeNameRegistry *result;
+
     if (!codepoints_by_name) {
         generate_codepoints_by_name(tc);
     }
     HASH_FIND(hash_handle, codepoints_by_name, cname, strlen((const char *)cname), result);
     MVM_free(cname);
+
     return result ? result->codepoint : -1;
 }
+/* XXX move this so it's included properly */
```

Comment:
fwiw, I think this particular routine would be better in `src/string/ops.c` and with a decl in `src/string/ops.h`.